streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -428,6 +428,68 @@ public void shouldUseCorrectDefaultsWhenNoneSpecified() {
         assertTrue(config.defaultTimestampExtractor() instanceof FailOnInvalidTimestamp);
     }
 
+    @Test(expected = StreamsException.class)
```

Comment:
We should not expect an exception here but insert a `fail(...)` after `config.keySerde()` within try-catch-block.
Same for all other test.